backend/private-graph/graph/schema.resolvers.go

```diff
@@ -1990,7 +1999,11 @@ func (r *mutationResolver) CreateErrorComment(ctx context.Context, projectID int
 
 			errorComment.Attachments = append(errorComment.Attachments, attachment)
 		} else if *s == modelInputs.IntegrationTypeJira {
-			if err := r.CreateJiraTaskAndAttachment(ctx, workspace, attachment, title, desc, *issueTeamID); err != nil {
+			if issueTeamID == nil || issueTypeID == nil {
```

Comment:
I noticed that `issueTeamID` is getting defaulted to an empty string in the request, whereas `issueTypeID` is defaulted to undefined. Is the case where it is an empty string a relevant edgecase?

Seems like these two variables should behavior similarly in the ts files, but not blocking

Reply:
I think changing `issueTeamID` to default to undefined makes sense since the backend has some logic to default in case it's `nil` but doesn't do anything to check the empty string, but it does look like there's some possible panics where we dereference it. Will take a look at this